tests/h/views/api/users_test.py

```diff
@@ -12,7 +12,26 @@
 from h.services.user_signup import UserSignupService
 from h.services.user_update import UserUpdateService
 from h.services.user_unique import UserUniqueService, DuplicateUserError
-from h.views.api.users import create, update
+from h.views.api.users import read, create, update
+
+
+class TestRead(object):
+    def test_it_presents_user(
+        self, pyramid_request, fakeUserContext, TrustedUserJSONPresenter
+    ):
+        read(fakeUserContext, pyramid_request)
+
+        TrustedUserJSONPresenter.assert_called_once_with(fakeUserContext.user)
+
+    def test_it_returns_presented_user(
+        self, pyramid_request, fakeUserContext, TrustedUserJSONPresenter
+    ):
+        result = read(fakeUserContext, pyramid_request)
+        assert result == TrustedUserJSONPresenter.return_value.asdict()
```

Comment:
Although this works in this case, we should be consistent about tests never calling mock methods themselves, as this adds to the mock's record of calls that have been made to it, which in many cases can be confusing or cause tests to not actually be testing what they should. Instead access `return_value`:

```suggestion
        assert result == TrustedUserJSONPresenter.return_value.asdict.return_value
```